pkg/descheduler/descheduler.go

```diff
@@ -366,88 +301,30 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 			eventRecorder,
 		)
 
-		var enabledDeschedulePlugins []enabledDeschedulePluginEntry
-		var enabledBalancePlugins []enabledBalancePluginEntry
-
-		// Build plugins
 		for _, profile := range deschedulerPolicy.Profiles {
-			pc, _ := GetPluginConfig(defaultevictor.PluginName, profile.PluginConfigs)
-			if pc == nil {
-				klog.ErrorS(fmt.Errorf("unable to get plugin config"), "skipping plugin", "plugin", defaultevictor.PluginName, "profile", profile.Name)
-				continue
-			}
-			evictorFilter, err := defaultevictor.New(
-				pc.Args,
-				&handleImpl{
-					clientSet:                 rs.Client,
-					getPodsAssignedToNodeFunc: getPodsAssignedToNode,
-					sharedInformerFactory:     sharedInformerFactory,
-				},
+			prfl, err := frwkprofile.NewProfile(
```

Comment:
much easier to read this way:
```suggestion
			currProfile, err := frameworkprofile.NewProfile(
```